assets/assets/utils/minio_utils.py

```diff
@@ -236,15 +236,19 @@ def upload_thumbnail(
     return True
 
 
-def delete_one_from_minio(bucket: str, obj: str, client: minio.Minio) -> bool:
+def delete_one_from_minio(
```

Comment:
could you rename it to something more generic, considering the fact we are not linked to minio

Reply:
Done